internal/strace/strace.go

```diff
@@ -47,8 +47,8 @@ var (
 	// TODO: We can see how we can potentially reuse regex patterns.
 	writePattern = regexp.MustCompile(`\S+ ([^,]+),.*`)
 
-	// Length of the hex prefix "0x".
-	hexPrefixLength = 2
+	// We expect bytes written in the write syscall to be in hex.
+	hexPrefix = "0x"
```

Comment:
nit: this can be declared outside the `var` block as `const`

Reply:
Done.